app/controllers/remote_search_controller.rb

```diff
@@ -1,8 +1,14 @@
 class RemoteSearchController < ApplicationController
   def show
+    @form = SearchForm.new(params[:search_form])
```

Comment:
We need to at least initialise the form object in `else` part of controller action, so all I can reasonably do is moving it few lines down.